AlgorithmFactory/QuantConnect.AlgorithmFactory.csproj

```diff
@@ -80,6 +84,10 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\Algorithm\QuantConnect.Algorithm.csproj">
+      <Project>{3240ACA4-BDD4-4D24-AC36-BBB651C39212}</Project>
+      <Name>QuantConnect.Algorithm</Name>
```

Comment:
Why did you need to add the algorithm to the algorithm factory project? Please also make sure to only reference the QuantConnect fork of pythonnet.